lib/init/config-initializer.js

```diff
@@ -292,6 +292,7 @@ function processAnswers(answers) {
             jsx: true
         };
         config.plugins = ["react"];
+        config.extends.push("plugin:react/recommended");
     } else if (answers.framework === "vue") {
         config.plugins = ["vue"];
         config.extends.push("plugin:vue/essential");
```

Comment:
Question for the team: if we decide to do this,  do we also want to use the recommended config for the Vue plugin?